conda/gpu-environment.yml

```diff
@@ -23,6 +23,7 @@ dependencies:
 - vadersentiment
 - transformers
 - pandas
+- python-facebook-api
```

Comment:
Not sure if this released on conda channel. If not then we need to add this in `-pip` block same as `presidio-analyzer`